lib/nuget-parser/parsers/dotnet-core-v2-parser.ts

```diff
@@ -110,11 +110,31 @@ function getRestoredProjectName(
   );
 }
 
+export function extractLocalPackages(libs: Record<string, any>): string[] {
```

Comment:
Comment with explanation or example if possible. Not sure I get it. Why if not serviceable and not sha512?